pkg/config/config_template.yaml

```diff
@@ -701,6 +701,15 @@ api_key:
 #
 # secret_backend_skip_checks: false
 
+## @param secret_backend_command_sha256 - string - optional
+## @env DD_SECRET_BACKEND_COMMAND_SHA256 - string - optional
+## `secret_backend_command_sha256` is the SHA256 of the backend command executable.
+## The executable for `secret_backend_command` must be an absolute path if `secret_backend_command_sha256` is used.
+##
+## For more information see: https://github.com/DataDog/datadog-agent/blob/main/docs/agent/secrets.md
+#
+# secret_backend_command_sha256: <SHA256>
```

Comment:
We might also want to document that the configuration file will be locked as it will prevent upgrading the configuration while the Agent it running.

On that subject, are we OK with that workflow ? Unless I'm misunderstanding the PR, we will no longer be able to update the configuration (with chef, puppet, ansible or other) while the agent is running.
This will for a larger "blind window" for our customer when editing the configuration.

Reply:
We are not locking the configuration file, we are locking the file specified by `secret_backend_command`.

Reply:
My bad :+1:

Reply:
One last detail: we should add that we're going to lock the `secret_backend_command` file, since it might prevent customer for updating it while the agent is running.